screens/UserAnnouncements.screen.jsx

```diff
@@ -0,0 +1,24 @@
+import { Dimensions, View } from 'react-native';
+import colors from 'themes/colors';
+import { useRoute } from '@react-navigation/native';
+import sizes from 'themes/sizes';
+import { AnnouncementsList } from './AnnouncementsList';
+
+export const UserAnnouncementsScreen = () => {
+  const route = useRoute();
+
+  return (
+    <View style={{
+      height: Dimensions.get('screen').height,
+      backgroundColor: colors.WHITE,
+      paddingVertical: sizes.PADDING_30,
```

Comment:
na górze w ios w liście jest za duży odstęp, powinien być mniej wiecej taki sam jak na dole (tam może być to ciezko widoczne bo header nawigacji nie ma box shadow wiec musisz go dodać, bo taka lista jest brzydka : P ) ale to już dla danego screena. W komponencie nawigacji jest gdzieś tam dodany ten box shadow albo po prostu dać mu żeby ten header sie wyświetlał

Reply:
wtedy wyrównać te paddingi

Reply:
na Androidzie te paddingi też są ogromne, trzeba je zmniejszyć